websites/N/Nitro Type/metadata.json

```
@@ -13,7 +13,7 @@
 		"nitrotype.com",
 		"www.nitrotype.com"
 	],
-	"version": "1.2.23",
+	"version": "1.3.23",
```

```suggestion
	"version": "1.2.24",
```

the change is so small in my opinion that a patch should be fine